isb_lib/sitemaps/sitemap_fetcher.py

```diff
@@ -13,14 +14,6 @@
 import json
 
 
```

Comment:
These broken forward class declarations broke mypy -- using the annotations fixed it.